stage2/02-net-tweaks/01-run.sh

```diff
@@ -25,7 +25,12 @@ network={
 EOL
 fi
 
-# Disable wifi on 5GHz models
+# Disable wifi on 5GHz models if WPA_COUNTRY is not set
 mkdir -p "${ROOTFS_DIR}/var/lib/systemd/rfkill/"
-echo 1 > "${ROOTFS_DIR}/var/lib/systemd/rfkill/platform-3f300000.mmcnr:wlan"
-echo 1 > "${ROOTFS_DIR}/var/lib/systemd/rfkill/platform-fe300000.mmcnr:wlan"
+if [ -v WPA_COUNTRY ]; then
```

Comment:
This returns an error for me: `[: -v: unary operator expected`

This more reliably detects whether `$WPA_COUNTRY` is set:

```bash
if [ -n "$WPA_COUNTRY" ]; then
```

Reply:
> This returns an error for me: [: -v: unary operator expected

Huh. I checked in zsh and bash and it worked in both. Anyway, I can change it to use `-n`.

Reply:
Fixed.